metron-interface/metron-rest/README.md

```diff
@@ -353,6 +355,20 @@ Request and Response objects are JSON formatted.  The JSON schemas are available
       * searchRequest - Search request
   * Returns:
     * 200 - Search results
+    
```

Comment:
This is not a problem for this PR, but rather the documentation in general.  The actual TYPES returned from the rest calls are not documented at all.